example/sysinfo/odp_sysinfo.c

```diff
@@ -399,6 +401,11 @@ int main(void)
 		return -1;
 	}
 
+	if (odp_dma_capability(&dma_capa)) {
```

Comment:
For completeness `odp_cls_capability()` and `odp_comp_capability()` could also be added.

Reply:
Added in v2